experimenter/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.test.tsx

```diff
@@ -100,20 +100,6 @@ describe("DirectoryColumnQA", () => {
       QA_STATUS_PROPERTIES[NimbusExperimentQAStatusEnum.GREEN].emoji,
     );
   });
-
-  it("renders nothing if qa status is not set", () => {
-    render(
-      <TestTable>
-        <DirectoryColumnQA
-          {...experiment}
-          qaStatus={NimbusExperimentQAStatusEnum.NOT_SET}
-        />
-      </TestTable>,
-    );
-    expect(screen.queryByTestId("directory-table-cell-qa")).toHaveTextContent(
-      "",
```

Comment:
can we update this with the following emoji?

Reply:
Better, I'll just parameterize the test on all the qa statuses 👍